lib/mri/pp.rb

```diff
@@ -419,7 +419,7 @@ def pretty_print_cycle(q) # :nodoc:
 class Data # :nodoc:
   def pretty_print(q) # :nodoc:
     q.group(1, sprintf("#<data %s", PP.mcall(self, Kernel, :class).name), '>') {
-      q.seplist(PP.mcall(self, Data, :members), lambda { q.text "," }) {|member|
+      q.seplist(PP.mcall(self, Kernel, :class).members, lambda { q.text "," }) {|member|
```

Comment:
This is because we don't define `Data#members` but define it on the `instance_methods_module`.
I don't think it makes much sense to define it on `Data` as it would be a megamorphic call to `members` on the actual object's class.
So rather I think we should upstream this change.

Reply:
https://github.com/ruby/pp/pull/20